benchmark/infrastructure/BenchmarkConfigurationShorthandVisitor.cpp

```diff
@@ -0,0 +1,65 @@
+// Copyright 2023, University of Freiburg,
+// Chair of Algorithms and Data Structures.
+// Author: Andre Schlegel (March of 2023, schlegea@informatik.uni-freiburg.de)
+
+#include "../benchmark/infrastructure/BenchmarkConfigurationShorthandVisitor.h"
+
+#include <any>
+
+#include "util/json.h"
+
+std::any ToJsonBenchmarkConfigurationShorthandVisitor::visitShortHandString(
+    BenchmarkConfigurationShorthandParser::ShortHandStringContext* context) {
+  return visitAssignments(context->assignments());
+}
+
+std::any ToJsonBenchmarkConfigurationShorthandVisitor::visitAssignments(
+    BenchmarkConfigurationShorthandParser::AssignmentsContext* context) {
+  nlohmann::json contextAsJson(nlohmann::json::value_t::object);
+
+  for (auto assignment : context->listOfAssignments) {
+    // Add the json representation of the assignment.
+    contextAsJson.update(
+        std::any_cast<nlohmann::json>(visitAssignment(assignment)));
```

Comment:
You could check here that the keys are distinct, s.t.
`a:42, a:43` throws an error.

Reply:
I had to take a quick look online, because I thought, key duplication was illegal in JSON, but that doesn't seem to be the case.
In that case, I will follow the standard and make the action non illegal.

Reply:
On second thought, could you please post the short hand string, that you gave to the program? ```update()``` should already be able to handle duplicated keys and `a:42, a:43` doesn't cause an exception on my system.

Reply:
There is now an explicit test for `a:42, a:43`, as of [da4b9b8](https://github.com/ad-freiburg/qlever/pull/972/commits/da4b9b863ccbcfcd5c805d5f5efb4320a1fc0ea5).

Reply:
Done, see [ce3819e](https://github.com/ad-freiburg/qlever/pull/972/commits/ce3819e8188b7f0f6c49eeb6914f78c100e73d90).